[Fundamentals] Migrating 'Understand Invoices' from KB (#4945)
cloudflare/cloudflare-docs: content/fundamentals/get-started/basic-tasks/account-maintenance/understand-invoices.md

---
pcx-content-type: how-to
title: Understand Cloudflare invoices
weight: 3
---

> ## Understand Cloudflare invoices

Reviewer: This should be an H1.

You will receive an invoice in the **Billing** section of your Cloudflare account when you:

>     * Change your Cloudflare plan type

Reviewer: You need to remove the indent from these for them to render correctly.

    * Upgrade or downgrade to or from a paid plan
    * Add a new domain to a Cloudflare account
    * Enable or renew a subscription or add-on service

> For any historical invoices not included in the **Billing** section, please [contact Cloudflare support](https://support.cloudflare.com/hc/en-us).

Reviewer: ```suggestion
For any historical invoices not included in the **Billing** section, [contact Cloudflare support](https://support.cloudflare.com/hc/articles/200172476).
```

{{<Aside type="note">}}

Enterprise customers receive invoices directly from the Cloudflare accounting team. 

{{</Aside>}}

---

## Enable email invoices from Cloudflare

To enable Cloudflare invoice emails, which are sent when you add or remove subscriptions from your account:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com), and select your account.

2. Select **Manage Account** > **Billing**.

3. Select **Invoices & Documents**.

4. In the **Billing email preferences** card, toggle the notification switch to **On**.

> Once enabled, you will receive an invoice via email: 

Reviewer: ```suggestion
Once enabled, you will receive invoices via email: 
```

> Once enabled, you will receive an invoice via email: 

Reviewer: Same comment about indentation.

* within one business day of initial set-up
* every month at the end of your billing period
* within one business day for all new purchases

## Download invoice

To download your Cloudflare invoice:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com), and select your account.

2. Select **Manage Account** > **Billing**.

3. Select **Invoices & Documents**.

4. Find the invoice you want to download and select the icon next to the invoice number.

{{<Aside type="note">}}

Invoices cannot be re-generated once issued. Any pending billing updates or changes appear in the next month's invoice.

{{</Aside>}}

---

## Troubleshooting your invoice

### Change in billing contact information

> If your billing contact information has changed, update your Cloudflare email address as soon as possible:

Reviewer: ```suggestion
If your billing contact information has changed, [update your Cloudflare email address](https://support.cloudflare.com/hc/articles/203471284#12345679) as soon as possible:
```
